wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginMeta.java

```diff
@@ -109,4 +119,85 @@ public PluginMeta attributes(Map<String, String> attributes) {
     this.attributes = attributes;
     return this;
   }
+
+  public PluginMeta category(String category) {
+    this.category = category;
+    return this;
+  }
+
+  public String getCategory() {
+    return category;
+  }
+
+  public PluginMeta publisher(String publisher) {
+    this.publisher = publisher;
+    return this;
+  }
+
+  public String getPublisher() {
+    return publisher;
+  }
+
+  public PluginMeta repository(String repository) {
+    this.repository = repository;
+    return this;
+  }
+
+  public String getRepository() {
+    return repository;
+  }
+
+  public PluginMeta tags(List<String> tags) {
+    this.tags = tags;
+    return this;
+  }
+
+  public List<String> getTags() {
+    return tags;
```

Comment:
Possible NPE. We do not return empty collections.

Reply:
fixed